tests/functional/api/BackbeatServer.js

```diff
@@ -30,19 +30,23 @@ function getUrl(options, path) {
     return `http://${options.host}:${options.port}${path}`;
 }
 
-function setHash(redisClient, fields, cb) {
-    const cmds = fields.map(field => {
+function setKey(redisClient, keys, cb) {
+    const cmds = keys.map(key => {
         // eslint-disable-next-line no-unused-vars
-        const [bucket, key, versionId, site] = field.split(':');
-        const value = getKafkaEntry(bucket, key, site);
-        return ['hset', REDIS_KEY_FAILED_CRR, field, value];
+        const [bucket, objectKey, versionId, site] = key.split(':');
+        const value = getKafkaEntry(bucket, objectKey, site);
+        return ['set', `${REDIS_KEY_FAILED_CRR}:${key}`, value];
```

Comment:
This is used because the test suite is not run with the `TEST_SWITCH` environment var. I updated the constant variable name to `TEST_REDIS_KEY_FAILED_CRR` to make that more apparent.

Reply:
Maybe we want to update that in the CI? I'm fine either way.